build/media_source/plg_system_shortcut/js/keyselect.es6.js

```diff
@@ -0,0 +1,89 @@
+		document.body.appendChild(modalDiv);
+		const keySelectModal = document.getElementById("keySelectModal");
+		Joomla.initialiseModal(keySelectModal);
+		keySelectModal.addEventListener('keydown', this.handleKeyDownEvent, false);
+		keySelectModal.addEventListener('keyup', this.handleKeyCombinationkeyUpEvent, false);
+		const saveKeyCombination = document.getElementById("saveKeyCombination");
+		saveKeyCombination.addEventListener('click', this.handleSaveCombinationkeyUpEvent, false);
+	}
+	setModalAttributes(element, attrs) {
+		for(const [key, value] of Object.entries(attrs)){
+			element.setAttribute(`${key}`, `${value}`);
+		}
+	}
+	handleKeySelectClickEvent(e){
+		e.preventDefault();
+		document.getElementById("currentKeyCombination").textContent = this.textContent;
+		document.getElementById("newKeyCombination").textContent = "";
+		document.getElementById("current_KeyEvent").value = this.getAttribute("data-class");
+		const keySelectModal = document.getElementById("keySelectModal");
+		window.bootstrap.Modal.getInstance(keySelectModal).show(keySelectModal);
+	}
+	handleKeyCombinationkeyUpEvent(e){
+		if(e.keyCode >= 65 && e.keyCode <= 90){
+			let keyValue = e.key;
+			keyValue = keyValue.toUpperCase();
+			document.getElementById("current_hasControl").value = 0;
+			document.getElementById("current_hasShift").value = 0;
+			document.getElementById("current_hasAlt").value = 0;
+			const newKeySelectCombination = [];
+			if(e.ctrlKey){
+				newKeySelectCombination.push("CTRL");
+				document.getElementById("current_hasControl").value = 1;
+			}
+			if(e.shiftKey){
+				newKeySelectCombination.push("SHIFT");
+				document.getElementById("current_hasShift").value = 1;
+			}
+			if(navigator.platform.match('Mac') ? e.metaKey : e.altKey){
+				newKeySelectCombination.push("ALT");
```

Comment:
i am not able to select` meta` key combinations

Reply:
http://www.lispworks.com/documentation/lw50/CLWUG-M/html/clwuser-m-206.htm